src/python/nxt_python_wsgi.c

```diff
@@ -692,10 +721,18 @@ static int
 nxt_python_add_sptr(nxt_python_ctx_t *pctx, PyObject *name,
     nxt_unit_sptr_t *sptr, uint32_t size)
 {
-    char      *src;
-    PyObject  *value;
+    char *src;
 
     src = nxt_unit_sptr_get(sptr);
+    return nxt_python_add_char(pctx, name, src, size);
```

Comment:
Two style issues: 1. variable declaration needs a double space between name and type; 2. missing empty line before return value.

I actually suggest to write it as one line (there's no need in an intermediate variable):
```c
return nxt_python_add_char(pctx, name, nxt_unit_sptr_get(sptr), size);
```